bazel/repositories.bzl

```diff
@@ -1372,12 +1372,13 @@ def _intel_dlb():
         build_file_content = """
 filegroup(
     name = "libdlb",
-    srcs = glob([
-        "dlb/libdlb/**",
-    ]),
+    srcs = glob(["dlb/libdlb/*"]),
     visibility = ["@envoy//contrib/dlb/source:__pkg__"],
 )
 """,
+        patch_args = ["-p1"],
+        patches = ["@envoy//bazel/foreign_cc:dlb.patch"],
+        patch_cmds = ["cp dlb/driver/dlb2/uapi/linux/dlb2_user.h dlb/libdlb/"],
```

Comment:
nit: consider using a symbolic link (`ln`) instead of copying.

Reply:
i would stick with copy -  bazel can be a bit unpredictable with symbolic links